src/data/compositors/cosmic.json

```diff
@@ -126,10 +150,6 @@
       "interface": "zwp_keyboard_shortcuts_inhibit_manager_v1",
       "version": 1
     },
-    {
-      "interface": "zwp_linux_dmabuf_v1",
-      "version": 5
-    },
```

Comment:
This looks fishy, cosmic supports dmabuf for sure, (probably the same goes for wl_drm).

Are you sure you have working GPU acceleration when generating this file?

Reply:
Oh yeah I don't have a GPU so I'll add it back.